docs/release-notes/NuGet-1.3.md

```diff
@@ -15,16 +15,6 @@ NuGet 1.3 was released on April 25, 2011.
 
 ## New Features
 
-### Streamlined Package Creation with symbol server integration
```

Comment:
This service no longer works.

Reply:
Team review: add a note that this service is no longer supported

Reply:
I added note : `The above is deprecated and no longer supported.`
Please review again.